lib/mbimport.js

```diff
@@ -189,20 +189,24 @@ var MBImport = (function () {
         appendParameter(parameters, 'annotation', release.annotation);
 
         // Label + catnos
-        for (let i = 0; i < release.labels.length; i++) {
-            let label = release.labels[i];
-            appendParameter(parameters, `labels.${i}.name`, label.name);
-            appendParameter(parameters, `labels.${i}.mbid`, label.mbid);
-            if (label.catno != 'none') {
-                appendParameter(parameters, `labels.${i}.catalog_number`, label.catno);
+        if (release.labels && Array.isArray(release.labels)) {
```

Comment:
```suggestion
        if (Array.isArray(release.labels)) {
```
This also works since `Array.isArray(undefined) === false`.

Reply:
Thanks @kellnerd, I added this improvement in forced pushed 137e8a55cd884a751c88d1b50ba8a32f87d77fb8. :)